lib/uffizzi/cli/connect.rb

```diff
@@ -204,7 +220,10 @@ def credential_readable_name(credential)
     def google_service_account_content(credential_file_path = nil)
       return ENV['GCLOUD_SERVICE_KEY'] if ENV['GCLOUD_SERVICE_KEY']
 
-      return Uffizzi.ui.say('Path to google service account key file wasn\'t specified.') if credential_file_path.nil?
+      if credential_file_path.nil?
+        Uffizzi.ui.say("Path to google service account key file wasn't specified.")
+        exit(1)
```

Comment:
this looks like it works exactly like `raise Uffizzi::Error.new("Path to google service account key file wasn't specified.") if credential_file_path.nil?`
why you decided to write like this?

Reply:
Just made it similar to another block with `exit`. Didn't know it would've been handled properly with `raise`.